components/brave_news/browser/feed_controller.cc

```diff
@@ -154,6 +154,12 @@ void FeedController::EnsureFeedIsUpdating(
                                            subscriptions)) {
                               VLOG(1) << "ParseFeed reported failure.";
                             }
+
+                             // Notify listeners of the updated feed
+                            for (auto& listener : controller->feed_listeners_) {
+                                  listener->OnFeedUpdated(controller->current_feed_.Clone());
```

Comment:
Does `OnFeedUpdated` exist? There seem to be some missing changes to the `mojom` and to the front end.

Also, this probably belongs in `NotifyUpdateDone`, so the other paths for finishing an update also notify the listeners